renku_notebooks/util/kubernetes_.py

```diff
@@ -150,8 +150,14 @@ def sort_conditions(conditions):
         return {"step": c.type, "message": c.message, "reason": c.reason}
 
     def get_pod_status(pod):
+        ready = True
         try:
-            ready = pod.status.container_statuses[0].ready
+            for status in pod.status.container_statuses:
+                ready = (
+                    ready
+                    and status.ready
+                    and getattr(pod.metadata, "deletion_timestamp", None) is None
```

Comment:
```suggestion
```

Reply:
The pod metadata is the same for every container, so suggesting to take it out of the loop.